.dockerignore

```diff
@@ -0,0 +1,3 @@
+yt/docker/*
```

Comment:
What is the point of this `.dockerignore` ?

Reply:
It ignores changes in `yt/docker`, so that yt would not be rebuilt after changes in dockerfiles